resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/submit/DriverConfigurationStepsOrchestrator.scala

```diff
@@ -71,6 +76,7 @@ private[spark] class DriverConfigurationStepsOrchestrator(
         .map(_.split(","))
         .getOrElse(Array.empty[String]) ++
         additionalMainAppPythonFile.toSeq ++
+        additionalMainAppRFile.toSeq ++
```

Comment:
Important here that, similar to Python Primary Resource, that the R File is distributed via --files.